app/controllers/application_controller/ci_processing.rb

```diff
@@ -116,6 +116,7 @@ def process_elements(elements, klass, task, display_name = nil, order_field = ni
     Rbac.filtered(klass.where(:id => elements).order(order_by)).each do |record|
       name = record.send(order_field.to_sym)
```

Comment:
Can this be moved under the `else` below?

Reply:
No, this is only for the request of policy deletion:
```
Delete a policy
---------------
CloudForms logs the following when a policy is deleted from the VMDB:
[----] I, [2017-03-03T13:26:34.193211] INFO -- : <AuditSuccess> MIQ(CiProcessing.process_element_destroy) userid: [test-user] - 
[0e9e8c56-ffab-11e6-8359-001a4aa0a7d6] Record deleted
Request that the audit line is amended to include the following:
- Name of the policy

```

Reply:
No, this is only for the request of policy deletion:
```
Delete a policy
---------------
CloudForms logs the following when a policy is deleted from the VMDB:
[----] I, [2017-03-03T13:26:34.193211] INFO -- : <AuditSuccess> MIQ(CiProcessing.process_element_destroy) userid: [test-user] - 
[0e9e8c56-ffab-11e6-8359-001a4aa0a7d6] Record deleted.

Request that the audit line is amended to include the following:
- Name of the policy
```